### DISCUSSION TITLE
Radio Station Public Page Designer

### DISCUSSION THREAD

**Discussion by JeffersonStarseed** [AzuraCast/AzuraCast#6926] [Ideas]
This would be a request to include an expansion for an easier way to add further-design or additional design-options to a public station page (or pages).  I know that this is already fully customizable using CSS or JS to use code to drop into Azuracast, but some of us don't even know how to approach utilizing this.  I've searched for a 3rd-party GUI/WYSIWYG-to-CSS or JS tool, and I'm failing finding something simple and useful for this (any suggestions anyone here would have would be welcome).

Anywayz, ...I noticed the last time this topic was brought-up was in 2020, so maybe could be a good time to bring it up again for discussion.

**Comment by gAlleb** [comment]
Some serious late 90s - early 2000s vibes I'm getting here:)

Let's see what other folks say. But as a last resort you can always embed `the Azuracast player` on any ready-to-use/template site. There are quite a few options out there.

**Comment by rogercreagh** [comment]
I also find the public page editing/customisation provided very limiting. Could we not at least have access to edit the html for the page and the ability to easily embed the schedule into the page.

I appreciate it is intended as a very simple single page front end for the station and that the better way to go is to embed to player into a separate site, but it does seem seriously limited and could be improved somewhat.

**Comment by punyauk** [comment]
Can I also add that, as far as I can work out, using the feature to add custom JS to public pages adds it in the body with no option to add in head. I've tried adding code for Matomo tracking and that fails, I'm guessing because they want it in the head section?  Would be great to have options for where custom stuff goes.
